### PR TITLE
Edit pass on intro.

### PR DESCRIPTION
**Pull request by jeromekelleher**
Light editing pass, removing a few words here and there.

### CODE REVIEW

File: popSimManu.tex [hunk 1]
```diff
@@ -137,25 +137,27 @@ \section*{Introduction}
...
 introduce methods that are validated by a set of bespoke simulations produced by individual
 groups. Beyond inefficiency from duplicated effort, such a situation leads
 to decreased reproducibility and transparency across the entire field.
 Here we describe \stdpopsim, a community-driven catalog
 of standardized population genetics models spanning a variety of common study organisms.
-This resource will empower the field to rigorously evaluate population genetic software
-and select the best tool for a given task, thereby substantially increasing the reliability of population genetic inferences.
+This resource will empower the field to rigorously evaluate population genetic software,
+%and select the best tool for a given task,
+thereby substantially increasing the reliability of population genetic inferences.
```
Comment (jeromekelleher) [inline popSimManu.tex:156]: I think the next paragraph explains this "choosing the best tool" bit quite well. The sentence read weirdly with it in, as there was no preparation for this idea of some tools being good in some cases and less good in others.

Comment (andrewkern) [reply popSimManu.tex:156]: agree

**Review by andrewkern — APPROVED** [review]
lgtm

**Comment by andrewkern** [comment]
ack this one got merged. sorry but should be fine.

**Comment by jeromekelleher** [comment]
It's good. We can probably merge here unless github specifically flags a conflict. No point in worrying about a linear history on this repo

**Pull request merged**